src/cloudflare/internal/test/d1/d1-api-test.wd-test

```diff
@@ -9,8 +9,9 @@ const unitTests :Workerd.Config = (
           (name = "worker", esModule = embed "d1-api-test.js"),
           (name = "d1-api-test-common", esModule = embed "d1-api-test-common.js"),
         ],
-        compatibilityDate = "2023-01-15",
-        compatibilityFlags = ["nodejs_compat"],
+        compatibilityDate = "2024-05-12",
+        compatibilityFlags = ["experimental", "nodejs_compat", "service_binding_extra_handlers", "streaming_tail_worker", "tail_worker_user_spans"],
```

Comment:
What is service_binding_extra_handlers needed for here? Same applies to src/cloudflare/internal/test/images/images-api-test.wd-test.
Also note that tail_worker_user_spans has been obsoleted, it is no longer needed to enable all of STW.